src/content-scripts/ActionMgr.js

```diff
@@ -36,7 +36,7 @@ class ActionMgr {
 				// removing duplicates
 				let links = Array.from(new Set(SelectedElements.Links.map((elem) => elem.href)));
 
-				if(e.ctrlKey) {
+				if(e.ctrlKey || Prefs.CopyLinksToClipboard) {
```

Comment:
It would be nice that `Prefs.CopyLinksToClipboard` copy links but if you press ctrl it opens them

Reply:
> It would be nice that Prefs.CopyLinksToClipboard copy links but if you press ctrl it opens them

To do it change the condition:
`if ((e.ctrlKey && !(Prefs.CopyLinksToClipboard)) || (!(e.ctrlKey) && Prefs.CopyLinksToClipboard)) {`

Reply:
I didn't run it to check but everything seems ok. Thank you